osrdyne/Dockerfile

```
@@ -0,0 +1,60 @@
+# syntax=docker/dockerfile:1
+
+##############
+# Cargo chef #
+##############
+FROM lukemathwalker/cargo-chef:0.1.67-rust-latest AS chef
```

`grcov` doesn't compile on latest version on rust >1.80.0 (see https://github.com/mozilla/grcov/pull/1191). We pinned the version of Rust in `editoast` and `gateway` in https://github.com/OpenRailAssociation/osrd/pull/8214.

```suggestion
FROM lukemathwalker/cargo-chef:0.1.67-rust-1.79.0 AS chef
```